messaging/src/main/java/org/axonframework/eventhandling/scheduling/dbscheduler/DbSchedulerScheduleToken.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2010-2022. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.eventhandling.scheduling.dbscheduler;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.github.kagkarlsson.scheduler.task.TaskInstanceId;
+import org.axonframework.eventhandling.scheduling.ScheduleToken;
+
+import java.beans.ConstructorProperties;
+
+import static java.lang.String.format;
+
+/**
+ * ScheduleToken implementation representing a scheduled DbScheduler Job.
+ *
+ * @author Gerard Klijs
+ * @since 4.8.0
+ */
+public class DbSchedulerScheduleToken implements ScheduleToken, TaskInstanceId {
```

Comment:
I would prefer a name like `DbSchedulerScheduledEventToken`. Just Schedule confused me.

Reply:
I understand, but it would be inconsistent with the other implementations, and it implements `ScheduleToken` not `ScheduledToken`.